screens/Map.js

```diff
@@ -73,7 +73,7 @@ function Map ({ navigation }) {
                 </MapView>
                 <Search testID="searchBar" navigation={navigation} />
                 <View style={styles.CurrentBuildingLocation}>
-                        <CurrentBuildingLocation />
+                        {/* <CurrentBuildingLocation /> */}
```

Comment:
Remove the import associated with CurrentBuildingLocation if it has to stay commented out.

Reply:
Not sure if it will remain commented out.

It will be done in the last refactoring PR if ever

Reply:
sounds good thanks !